hedera-node/src/main/resources/bootstrap.properties

```diff
@@ -72,6 +72,11 @@ scheduling.whitelist=ConsensusSubmitMessage,CryptoTransfer
 tokens.maxPerAccount=1000
 tokens.maxSymbolUtf8Bytes=100
 tokens.maxTokenNameUtf8Bytes=100
+nfts.maxMetadataBytes=100
+nfts.maxBatchSizeBurn=100
+nfts.maxBatchSizeWipe=100
+nfts.maxBatchSizeMint=100
```

Comment:
Lets change the default properties to 10 instead of 100. For transfers, we have limit of 10 token transfers, so lets have the same value here as-well. It will be adjusted in time but lets try with 10 initially